test/avocado/seleniumlib.py

```diff
@@ -82,9 +85,9 @@ def tearDown(self):
             try:
                 # use time.clock() to ensure that snapshot files are unique and ordered
                 # sample name is like: screenshot-teardown-172434.png
-                screenshot_file = "screenshot-teardown-%s.png" % str(time.clock())[2:]
+                screenshot_file = "screenshotTeardown%s.png" % str(time.clock())[2:]
 
-                self.driver.save_screenshot(screenshot_file)
+                self.driver.save_screenshot(os.path.join(str(actualpath),screenshot_file))
```

Comment:
It is there for sure :-) I'm not sure if this method direcly return str, and not some other file object